CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
   button in the stack header for showing the context menu
 * Added `GridAutosizeMode.MANAGED` to autosize Grid columns on data or `sizingMode` changes, unless
   the user has manually modified their column widths.
+* Copying from Grids to the clipboard will now use the formatting provided by the `exportValue` and
```

Comment:
I don't think this is correct since we will never apply the `exportFormat` on the value we put in the clipboard (nor can we since it is an excel-specific format string in most cases)

Reply:
we do apply in the most common cases.  Understand you point bug going to let this stand..